src/m365/spo/commands/list/list-roleinheritance-reset.ts

```diff
@@ -65,6 +70,15 @@ class SpoListRoleInheritanceResetCommand extends SpoCommand {
           return isValidSharePointUrl;
         }
 
+        const listOptions: any[] = [args.options.listId, args.options.listTitle, args.options.listUrl];
```

Comment:
Let's not check this manually and instead use option sets with all three options.